app/components/c-product-detail.vue

```diff
@@ -26,7 +26,10 @@
     <section :class="b('bottom')">
 
       <div :class="b('main', { area: 'bottom' })">
-        <div :class="b('details')"> details</div>
+        <div v-if="product.description" :class="b('details')">
```

Comment:
- [ ] As I understand the `details` section is not only for `product.description` There are also the accordions with "technical details", documents etc. So I would use the `details` as a wrapper for this components